app/views/static_pages/landing_page.html.erb

```diff
@@ -23,59 +23,59 @@
     </p>
     <p class="govuk-body">
       If you want to know more about the qualifying criteria, check our 
-      <a class="govuk-link" href="/">eligibility page.</a>
+      <a class="govuk-link" href="/">eligibility page</a>.
```

Comment:
I think the eligibility page needs updating? Pretty certain its in master now so possibly just a rebase

Reply:
I've updated the link